politeiawww/proposals.go

```diff
@@ -405,6 +405,56 @@ func convertWWWPropCreditFromDatabasePropCredit(credit user.ProposalCredit) www.
 	}
 }
 
+// updatePropPointersWithVoteSummary takes a Proposal Record
+// and updates it with a summary of its voting process.
+func (p *politeiawww) updatePropWithVoteSummary(prop *www.ProposalRecord) error {
+	props := make([]*www.ProposalRecord, 0, 1)
+	props = append(props, prop)
+	return p.updatePropPointersWithVoteSummary(props)
+}
+
+// updatePropPointersWithVoteSummary takes a slice of Proposal Records
+// and updates them with a summary of their voting process.
+func (p *politeiawww) updatePropsWithVoteSummary(props []www.ProposalRecord) ([]www.ProposalRecord, error) {
+	propPointers := make([]*www.ProposalRecord, len(props))
+	for i := range props {
+		propPointers[i] = &props[i]
+	}
+
+	err := p.updatePropPointersWithVoteSummary(propPointers)
+	if err != nil {
+		return nil, err
+	}
+
+	return props, nil
+}
+
+// updatePropPointersWithVoteSummary takes a slice of pointers to Proposal
+// Records and updates them with a summary of their voting process.
+func (p *politeiawww) updatePropPointersWithVoteSummary(props []*www.ProposalRecord) error {
```

Comment:
Minor style nit.  In general, the politeia repo will list functions from the bottom up, meaning the calling functions are below the functions they call.  This goes against what might be considered idiomatic go, but it makes the repo easier to read if everything is consistent.  Can you move `updatePropPointersWithVoteSummary` above `updatePropsWithVoteSummary`?